synfig-studio/src/gui/asyncrenderer.h

```diff
@@ -77,6 +82,9 @@ class AsyncRenderer : public etl::shared_object, public sigc::trackable
 	//! Current rendering status
 	//! Undefined if not finished
 	Status status;
+        
+	//! Handles user interaction on the rendering process
+	Interaction interaction;
```

Comment:
See below Why this should be deleted.

Reply:
Here you can find my compilation output: https://pastebin.com/xjaBNZL7